### PR TITLE
[RHACS] Post-GA 3.72 Change Requests

### PR DESCRIPTION
**Pull request by kcarmichael08**
Feedback for Release Notes and other doc changes that didn't make it in before GA on 9/26:
- change "inactive" to nonactive
- change "deletion" to "removal from RHACS"
- additional misc. feedback on Release Notes

Versions:

- `rhacs-docs-3.72`
- `rhacs-docs`

Issue: none (tracked in Google doc)

Link to docs preview:

- [Release Notes](https://openshift-docs-k4uzg0jvy-kcarmichael08.vercel.app/openshift-acs/master/release_notes/372-release-notes.html)
- [Configuring automatic removal of nonactive clusters from RHACS](https://openshift-docs-k4uzg0jvy-kcarmichael08.vercel.app/openshift-acs/master/configuration/configure-inactive-cluster-deletion.html#configure-active-cluster-deletion)
- [Configure cluster decommissioning](https://openshift-docs-k4uzg0jvy-kcarmichael08.vercel.app/openshift-acs/master/configuration/configure-inactive-cluster-deletion.html#configure-cluster-decommissioning_configure-nonactive-cluster-removal)
- [Identifying Dockerfile lines in images that introduced components with CVEs](https://openshift-docs-k4uzg0jvy-kcarmichael08.vercel.app/openshift-acs/master/operating/examine-images-for-vulnerabilities.html#identify-dockerfile-line-component-cve_examine-images-for-vulnerabilities)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: release_notes/372-release-notes.adoc [hunk 1]
```diff
@@ -154,21 +173,23 @@ In the table, features are marked with the following statuses:
...
+. `/v1/cves/suppress` and `/v1/cves/unsuppress` have been deprecated and will be removed in a future release. After these are removed:
 ** Use `/v1/imagecves/suppress` and `/v1/imagecves/unsuppress` to snooze and unsnooze image vulnerabilities.
 ** Use `/v1/nodecves/suppress` and `/v1/nodecves/unsuppress` to snooze and unsnooze node and host vulnerabilities.
 ** Use `/v1/clustercves/suppress` and `/v1/clustercves/unsuppress` to snooze and unsnooze platform (Kubernetes, Istio, and {ocp}) vulnerabilities.
-* Permission `ClusterCVE` is deprecated and will be superseded by the existing permission `Cluster`. 
-* `Label` and `Annotation` search options in {product-title-short} are deprecated and will be removed in the {product-title-short} 3.73 release. They will be replaced by the search options listed in the following table.
-
+. Previously, groups were retrieved by using the field `props`: `props.authProviderId`, `props.key`, and `props.value`. This field will be replaced by the new `props.id` field. Use the `props.id` field to retrieve groups in the {product-title-short} API. Note the following:
+** Retrieval by using the `props` fields will be removed in version a future release.
```
Comment (0x656b694d) [inline release_notes/372-release-notes.adoc:183]: "_in version a future release_"?

**Comment by theencee** [comment]
Why go from inactive to nonactive?

File: release_notes/372-release-notes.adoc (hunk 1 above)
Comment (kcarmichael08) [reply release_notes/372-release-notes.adoc:183]: aaaaargh, thank you

**Comment by kcarmichael08** [comment]
> Why go from inactive to nonactive?

That was changed in the original RN by @rhdcaspin & other PM feedback - I believe because "inactive" had a meaning that wasn't wanted to be used here for clusters that were not communicated with and subject to removal. I didn't have time to rename all the files, etc. so this PR is to do that, but the RNs already use "nonactive" per their feedback.

File: modules/view-nonactive-clusters.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+[id="view-nonactive-clusters_{context}"]
+= Viewing nonactive clusters
+
+.Procedure
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *System Configuration*.
+. In the *Cluster deletion* section, click *Clusters which have Sensor Status: Unhealthy* to go to the *Clusters* list page. This page is filtered to show nonactive clusters that are subject to removal from {product-title-short} and a timeframe for the removal. 
++
+[NOTE]
+====
+If this feature is enabled after a cluster is considered nonactive, the counting of days towards removal starts from the time the cluster became nonactive, not from the time that the feature was enabled. If there are any nonactive clusters that you do not want removed, you can configure a label as described in the "Configuring cluster decommissioning" section. Labeled clusters are ignored when the system removes nonactive clusters.
```
Comment (theencee) [inline modules/view-nonactive-clusters.adoc:14]:
```suggestion
If this feature is enabled after a cluster is considered nonactive, the counting of days towards removal starts from the time the cluster became nonactive, not from the time that the feature was enabled. If there are any nonactive clusters that you do not want removed, you can configure a label as described in the "Configuring cluster decommissioning" section. Clusters with those labels are not removed even if they are nonactive.
```

Something like that? The current wording made me think all labelled clusters are not removed which is incorrect. Only the ones with a label that are specified.

**Review by theencee — APPROVED** [review]

**Review by parametalol — APPROVED** [review]

**Review by sheriff-rh — APPROVED** [review]
Looks really great, Kerry! I have no recommendations for improvement.

File: modules/view-nonactive-clusters.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * configuration/cluster-decommissioning.adoc
+:_content-type: PROCEDURE
+[id="view-nonactive-clusters_{context}"]
+= Viewing nonactive clusters
+
```
Comment (kalexand-rh) [inline modules/view-nonactive-clusters.adoc:7]: You need an abstract here.

**Review by kalexand-rh — COMMENTED** [review]
I have one request, which I know is only showing up as a change because of the file rename, but I think it's worth adding.

File: modules/view-nonactive-clusters.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+[role="_abstract"]
+Nonactive clusters are clusters that were installed and performed a handshake with Central at least once, but have been unreachable by Sensor for a specified period of time. Use this procedure to view a list of these clusters.
```
Comment (kalexand-rh) [inline modules/view-nonactive-clusters.adoc:9]:
```suggestion
Nonactive clusters are clusters that were installed and performed a handshake with Central at least once but have been unreachable by Sensor for a specified period of time. Use this procedure to view a list of these clusters.
```

**Pull request merged**